Resources/public/js/views/ez-languageselectionboxview.js

```diff
@@ -0,0 +1,278 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-languageselectionboxview', function (Y) {
+    "use strict";
+    /**
+     * Provides the language selection box view class
+     *
+     * @method ez-languageselectionboxview
+     */
+    var CONFIRM = 'confirm',
+        CANCEL = 'cancel';
+
+    /**
+     * The language selection box view.
+     *
+     * @namespace eZ
+     * @class LanguageSelectionBoxView
+     * @constructor
+     * @extends eZ.TemplateBasedView
+     */
+    Y.eZ.LanguageSelectionBoxView = Y.Base.create('languageSelectionBoxView', Y.eZ.TemplateBasedView, [], {
+        TRANSLATION_TPL: '<li class="ez-translation" />',
```

Comment:
I tried it, but while I was basing on confirmBox and UDW the view is loaded on start and I have to fill it with languages dynamically. Or maybe there is another to reload the view?